src/main/java/com/iota/iri/storage/rocksDB/RocksDBPersistenceProvider.java

```diff
@@ -318,6 +319,29 @@ public boolean saveBatch(List<Pair<Indexable, Persistable>> models) throws Excep
         }
     }
 
+    public void deleteBatch(Collection<? extends Indexable> keys, Class<? extends Persistable> type)
```

Comment:
I prefer these names - but they are not consistent w/ the rest of the `RocksDBPP`.
this makes for a confusing API.

Reply:
changed to the same names saveBatch.
Also, similarly to SaveBatch I also don't use the `?` to allow the collection to be heterogeneous.